bulbs/instant_articles/templates/instant_article/base_instant_article.html

```diff
@@ -38,7 +38,7 @@ <h3 class="op-kicker">
 
         <figure>
           {% block cover %}
-            <img src="{% cropped_url content.detail_image.id %}" />
+            <img src="{% cropped_url content.detail_image.id fixed=True %}" />
```

Comment:
We should add a Tech Debt task to rename "fixed" and "BETTY_FIXED_URL" to something more expected like "force_public" and "BETTY_PUBLIC_IMAGE_URL`